src/main/io/gps.c

```diff
@@ -843,28 +1373,37 @@ void gpsUpdate(timeUs_t currentTimeUs)
 
     // read out available GPS bytes
     if (gpsPort) {
+        DEBUG_SET(DEBUG_GPS_CONNECTION, 7, serialRxBytesWaiting(gpsPort));
+        static uint8_t wait = 0;
+        static bool isFast = false;
         while (serialRxBytesWaiting(gpsPort)) {
-            if (cmpTimeUs(micros(), currentTimeUs) > GPS_MAX_WAIT_DATA_RX) {
-                // Wait 1ms and come back
-                rescheduleTask(TASK_SELF, TASK_PERIOD_HZ(TASK_GPS_RATE_FAST));
+            int bytes_processed = 0;
```

Comment:
`int bytes_processed = 0;` must be outside of `while(serialRxBytesWaiting(gpsPort))`

Reply:
Strangely, it worked as it was.  But I've put it before the while() function, as sugested.  Thanks.

Reply:
ok done